releaser-core/src/main/java/releaser/internal/project/ProjectCommandExecutor.java

```diff
@@ -166,6 +187,35 @@ ReleaserProcessExecutor executor(String workDir) {
 		return new ReleaserProcessExecutor(workDir);
 	}
 
+	public void publishAntoraDocs(File antoraDocsProject, File project, ReleaserProperties properties) {
+		try {
+			copyRunnerToActions(antoraDocsProject);
+			String command = new CommandPicker(properties).publishAntoraDocsCommand(project, properties);
+			log.info("Executing command for publishing Antora docs " + command + " / " + properties);
+			String[] commands = command.split(" ");
+			// TODO fix this hack
+			for (int i = 0; i < commands.length; i++) {
```

Comment:
What's this? :P

Reply:
Ah yes.  I was having an issue where if I put the host key in the command before calling `.split` it was messing up the command because the host key has line breaks and spaces.  This was the only way I could think of making it work at the moment, but I am open to other ideas if you have them.

Reply:
OH I see. Yeah that's a good way to go then